packages/terra-button-group/src/ButtonGroup.module.scss

```diff
@@ -27,12 +27,17 @@
       opacity: var(--terra-button-group-button-disabled-opacity, 0.3);
     }
 
+    &:focus {
+      outline: var(--terra-button-group-button-focus-outline, 2px dashed #000); 
```

Comment:
this change is more appropriate on button component not on button group. As button-group consumes button focus styles should be updated on button. we need make sure outline color are appropriate for different button variants.

Reply:
Done.